src/routes/Login.jsx

```diff
@@ -0,0 +1,4 @@
+const Login = () => {
+  return <div>Login page</div>;
```

Comment:
You did a great job! so far but here contents of the login page are not present please make a login page as it is required to login and edit the task otherwise edit button should be disabled and also you need to add a context hook for authentication. Please make these changes and submit them again for review. If you need any clarification please mention me @fpsapc in a comment, so I can get a notification.